rasa/core/schemas/domain.yml

```diff
@@ -27,6 +27,33 @@ mapping:
   responses:
     type: "map"
     allowempty: True
+    matching-rule: 'any'
```

Comment:
not necessary since there's only one regex below.

Reply:
yeah and it's the default anyway